### PR TITLE
docs(tpch): add prefix 'format_' for settings used in streaming_load.

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Fixes https://github.com/datafuselabs/databend/issues/8094

### CODE REVIEW

**Comment by xudong963** [comment]
If need to update other places in doc that use `field_delimiter`? Or only streaming_load needs `format_` prefix?

**Comment by youngsofun** [comment]
used in both clickhouse hander and streaming load.
`docs` already updated, forgot `website/`

File: website/blog/2022-08-08-benchmark-tpc-h.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ The code below connects to Databend using the Root user. Please note that the ro
 for t in customer lineitem nation orders partsupp part region supplier
 do
     echo "$t"
-    curl -XPUT 'http://root:@127.0.0.1:8000/v1/streaming_load' -H 'insert_sql: insert into tpch.'$t' format CSV' -H 'skip_header: 0' -H 'field_delimiter:|' -H 'record_delimiter: \n' -F 'upload=@"./'$t'.tbl"'
```
Comment (xudong963) [inline website/blog/2022-08-08-benchmark-tpc-h.md:159]: why `'skip_header: 0'` and `'record_delimiter: \n'` are deleted?

Comment (youngsofun) [reply website/blog/2022-08-08-benchmark-tpc-h.md:159]: these are default values

**Review by xudong963 — APPROVED** [review]

File: website/blog/2022-08-08-benchmark-tpc-h.md (hunk 1 above)
Comment (youngsofun) [reply website/blog/2022-08-08-benchmark-tpc-h.md:159]: add back `-H 'record_delimiter: \n' ` since the default is `\r\n`

**Pull request merged**